packages/salesforcedx-vscode-core/src/context/workspaceOrgType.ts

```diff
@@ -22,17 +22,8 @@ export async function getWorkspaceOrgType(
     e.name = 'NoDefaultusernameSet';
     throw e;
   }
-
   const username = await OrgAuthInfo.getUsername(defaultUsernameOrAlias);
-
-  if (isNullOrUndefined(username)) {
-    telemetryService.sendException(
-      'get_workspace_org_type',
-      'workspaceOrgType.getWorkspaceOrgType ran into an undefined username.'
-    );
-  }
-
-  const isScratchOrg = await OrgAuthInfo.isAScratchOrg(username!).catch(err =>
+  const isScratchOrg = await OrgAuthInfo.isAScratchOrg(username).catch(err =>
```

Comment:
Why was "username!" changed to "username" here?

Reply:
We don't need to force remove (`undefined|null`), since there isn't a way for it to be `undefined` here.